syncstorage/tests/functional/test_storage.py

```diff
@@ -1485,7 +1484,6 @@ def test_batch_ttl_update(self):
         self.assertEquals(len(res), 1)
         self.assertEquals(res[0]['payload'], 'see')
 
-    @unittest2.skip("for the moment")
     def test_batch_ttl_is_based_on_commit_timestamp(self):
```

Comment:
That's cool!  Part of the reason we use real times in these tests, is that we also run them against a live server to test for spec conformance.  But it may be possible to use the freezegun approach when running locally, and only sleep when running against a remote server.